### PR TITLE
Code Sandbox action button layout updates

### PR DESCRIPTION
**Pull request by dbchristopher**
## :white_check_mark: Requirements

- [x] Includes test coverage for all changes
- [x] Build and tests are passing
- [x] PR is ideally < 400LOC

## :sparkles: Changes

- Update layout for code sandbox actions, so as not to unintentionally obscure any of the code samples
- Updates Tooltip documentation to demonstrate placement options. 

## :camera: Screenshots

sandbox-actions

### CODE REVIEW

File: packages/www/src/MDX/Pre/CodeSandbox.tsx [hunk 1]
```diff
@@ -205,16 +208,12 @@ const ActionButton = styled(IconButton)<ActionProps>`
 `
 
 const ActionLayout = styled.div<ActionProps>`
-  transition: background 0.3s;
+  transition: background 0.35s;
```
Comment (lukelooker) [inline packages/www/src/MDX/Pre/CodeSandbox.tsx]: Those 5 hundredth's of a second make all the difference 😛

File: packages/www/src/documentation/components/overlays/tooltip.mdx [hunk 2]
```diff
@@ -33,3 +33,56 @@ A simple Tooltip component with out of the box styles and behavior.
   )}
 </Tooltip>
 ```
+
+## Tooltip Placement
```
Comment (lukelooker) [inline packages/www/src/documentation/components/overlays/tooltip.mdx]: Nice! We should build a sandbox component for this eventually - a la: https://codesandbox.io/s/uxf2l

**Comment by lukelooker** [comment]
Oh - minor comment. Please add a CHANGELOG update for this PR.

File: packages/www/src/MDX/Pre/CodeSandbox.tsx (hunk 1 above)
Comment (dbchristopher) [reply packages/www/src/MDX/Pre/CodeSandbox.tsx]: 😂 what can I say

**Pull request merged**